### ISSUE TITLE
Reading: Time Waits for No One! Analysis and Challenges of Temporal Misalignment

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#259]
## 0. Paper
 - paper: [link](https://aclanthology.org/2022.naacl-main.435/)
 - NAACL2022

## 1. What is it?
They analyzed the effect of temporal misalignment in downstream tasks.
 - effect of domain/temporal adaptation (additional training) on target corpus
 - effect of fine-tuning

![スクリーンショット 2023-05-07 9 28 31](https://user-images.githubusercontent.com/45454055/236651754-03a1793b-6667-40d1-a161-9f39843af852.png)

## 2. What is amazing compared to previous works?
They conduct a wide range of analyses in temporal misalignment (eight tasks).

![スクリーンショット 2023-05-07 9 29 21](https://user-images.githubusercontent.com/45454055/236651772-5b85e113-8a99-4469-a8cf-d44fd0430af4.png)

## 3. Where is the key to technologies and techniques?
They proposed a metric for evaluating the speed of degradation.
![スクリーンショット 2023-05-07 9 49 35](https://user-images.githubusercontent.com/45454055/236652252-8a742c1c-99ef-4e14-9f5f-0d29702a295c.png)

![スクリーンショット 2023-05-07 9 55 42](https://user-images.githubusercontent.com/45454055/236652371-8ed10d58-35d3-4877-bb0c-33f1ab39a25d.png)

Figure 2 shows that the performance degradation occurs linearly.

### 4.1 Temporal Misalignment
![スクリーンショット 2023-05-07 9 56 20](https://user-images.githubusercontent.com/45454055/236652381-0b3e28f3-3e90-4e38-a862-e8182d48a387.png)

Figure 4 shows that performance degradation strongly occurs on Twitter (PoliAff), not on Scientific papers (SciERC) and Yelp (YelpCls)

### 4.2 Effect of domain/temporal adaptation and finetuning (Figure 1)

![スクリーンショット 2023-05-07 9 58 59](https://user-images.githubusercontent.com/45454055/236652440-424d7a0f-5625-4eaf-8761-97a9af820e2b.png)

Table 3 shows that  

$$ Temporal\ adaptation\ (additional\ training) < Finetuning $$
